repo/packages/A/avi/0/config.json

```diff
@@ -0,0 +1,197 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
there is a relatively new requirement in dcos for the framework-name to be  service.name.      The marathon app-id name is based on this... currently this is deployed with the name `avi-default` to avoid `-default` being added we need the configuration similar to this example: https://github.com/mesosphere/universe/blob/kerberos-hdfs/repo/packages/K/kdc/0/config.json#L5-L9